TLM/TLM/Resources/lang.txt

```diff
@@ -59,7 +59,7 @@ All_vehicles_may_ignore_lane_arrows All vehicles may ignore lane arrows
 Busses_may_ignore_lane_arrows Buses may ignore lane arrows
 Reckless_driving Reckless driving
 TMPE_Title Traffic Manager: President Edition
-Settings_are_defined_for_each_savegame_separately Settings are defined for each savegame separately
+Settings_are_defined_for_each_savegame_separately Settings are defined for each savegame separately. Please open settings while in-game.
```

Comment:
Is there a point of adding this line? I feel as if this is strongly implied and people can figure that out.

Reply:
I think it's needed for disambiguation, especially has "read only mod options" (when accessed from main menu) is such a weird concept considering how all other mods work. The original text made it sound like global settings would still have an effect but the save game takes precedence, which is not what happens.

IMO, better text might be something like:

> Settings can only be changed whilst in-game, and are unique to each save game.

Reply:
Changed to the suggested.

Reply:
The changed version works much better.